sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/optimizer/AggregateOptimizeSuite.scala

```diff
@@ -34,18 +34,27 @@ class AggregateOptimizeSuite extends PlanTest {
 
   object Optimize extends RuleExecutor[LogicalPlan] {
     val batches = Batch("Aggregate", FixedPoint(100),
+      FoldablePropagation,
       RemoveLiteralFromGroupExpressions,
       RemoveRepetitionFromGroupExpressions) :: Nil
   }
 
+  val testRelation = LocalRelation('a.int, 'b.int, 'c.int)
+
   test("remove literals in grouping expression") {
-    val input = LocalRelation('a.int, 'b.int)
+    val query1 = testRelation.groupBy('a, Literal("1"), Literal(1) + Literal(2))(sum('b))
+    val query2 = testRelation.groupBy('a)(sum('b))
+    val optimized = Optimize.execute(analyzer.execute(query1))
+    val correctAnswer = analyzer.execute(query2)
```

Comment:
please follow the code style for this file:
```
val query = ...
val optimized = ...
val correctAnswer = ...
```
`query1` and `query2` looks weird here.